src/assembler.jl

```diff
@@ -58,7 +58,8 @@ end
 Assembles the element residual `ge` into the global residual vector `g`.
 """
 function assemble!{T}(g::AbstractVector{T}, ge::AbstractVector{T}, edof::AbstractVector{Int})
-    for i in 1:length(edof)
+    @boundscheck checkbounds(g, edof)
```

Comment:
IIUC this `checkbounds` call will be skipped if this function is called with `@inbounds assemble!(g, ge, edof)` ?